src/test/subsidy_tests.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2014-2015 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// Copyright (c) 2014-2017 The Dash Core developers
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
This should probably say
```
+// Copyright (c) 2014-2017 The Dash Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

(1. No Bitcoin Core 2. Comment message was messed up by some previous changes)